src/plcc/load_spec/parse_spec/parse_semantic_spec/parse_codeFragments.py

```diff
@@ -0,0 +1,71 @@
+from dataclasses import dataclass
+from .parse_targetLocator import TargetLocator, parse_targetLocator
+from plcc.load_spec.load_rough_spec.parse_lines import Line
+from plcc.load_spec.load_rough_spec.parse_blocks import Block
+from plcc.load_spec.load_rough_spec.parse_dividers import Divider
+import re
+
+@dataclass
+class CodeFragment:
+    targetLocator: TargetLocator
+    block: Block
+
+def parse_codeFragments(lines_and_blocks: list[Line | Block]):
```

Comment:
Please rename this function and the containing file/module parse_code_fragments.